packages/notifi-web-push-service-worker/lib/utils/service-worker.ts

```diff
@@ -97,6 +98,15 @@ export const isNotifiWebPushEventData = (
   const eventData = data as NotifiWebPushEventData;
   if (!('Subject' in eventData)) return false;
   if (!('Message' in eventData)) return false;
-  // if (!('EncryptedBlob' in eventData)) return false; // TODO: uncomment when BE is ready
+  if (!('EncryptedBlob' in eventData)) return false;
```

Comment:
I think we still not yet have the `'EncryptedBlob'` in the payload, which could break the app (I tested on prod):
![Screenshot 2024-09-17 at 22 15 16](https://github.com/user-attachments/assets/873f0f39-953d-46a8-8532-537d16b80919)